tests/cases/helpers/case_helpers_care_platform.yaml

```diff
@@ -1714,4 +1714,26 @@ CarePartnerEditParticipantNavigation:
       Strategy: xpath
       Id: $PAGE.care_platform_participant_records.edit_info_button$
     - Type: sleep
-      Time: 1
\ No newline at end of file
+      Time: 1
+
+CarePartnerToastifyValidation:
```

Comment:
it would be good to use a name that is more business related like 'Toast', Toastify is the library used for the toasts and it may be confusing in the future for someone that doesn't know that.

Reply:
Yes, I changed to "CarePartnerToastValidation"